guide/src/function.md

```diff
@@ -96,6 +96,55 @@ The `#[pyo3]` attribute can be used to modify properties of the generated Python
         m.add_function(wrap_pyfunction!(pyfunction_with_module, m)?)
     }
     ```
+  - <a id="deprecated"></a> `#[pyo3(deprecated(message = "...", category = ...))]`
+
+    Set this option to mark the function as deprecated. The `message` parameter is a string that will be displayed when the function is called, and the `category` parameter is a string that will be displayed in the deprecation warning. The `category` parameter is optional and has to be a subclass of [`Warning`](https://docs.python.org/3.12/library/exceptions.html#Warning). When the `category` parameter is not provided, the warning will be defaulted to [`DeprecationWarning`](https://docs.python.org/3.12/library/exceptions.html#DeprecationWarning).
```

Comment:
`` `category` parameter is a string that will be displayed in the deprecation warning ``

I assume this is some kind of mixup with `message`, at least this does not make a lot of sense to me like this.